lib/board.rb

```diff
@@ -10,9 +12,11 @@ class Board
   def initialize(inOut, board_size = 3)
     @inOut = inOut
     @board = []
```

Comment:
Still feels like you are tracking a lot of array state here. Can anything be simplified?